source/code-of-conduct/index.html.haml

```diff
@@ -1,44 +1,39 @@
 ---
-title: Brug - Code of Conduct - Ruby Belgium
+title: Code of Conduct - Ruby Belgium
 ---
 
-%header.header
+%header
```

Comment:
I did not say cumbersome… Less predictable yes because alternate can mean different things in different context and you don’t want to worry about inheriting some unwanted properties.  You don’t want to over specify your selectors as well:

<img width="978" alt="screenshot 2017-01-01 22 44 37" src="https://cloud.githubusercontent.com/assets/39081/21583160/2dda8bd0-d074-11e6-8114-322f98deff6f.png">

From Mozilla https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Writing_efficient_CSS



You’re not correct when you say ```.section--alternate``` increases the specificity. It’s the opposite.
<img width="686" alt="screenshot 2017-01-01 22 40 10" src="https://cloud.githubusercontent.com/assets/39081/21583142/4a7c68b8-d073-11e6-9a9e-192ad3169a5e.png">

You can indeed regroup common props under alternate but then I’d say the name should be more descriptive.

You can even go further and avoid this particular case by using the ```&:odd``` and even ```&:even``` css3 selector